### PR TITLE
- added platform diff when looking for aws creds folder

### PR DESCRIPTION
**Pull request by AndreyBulezyuk**
## What changes are proposed in this pull request?

(Please fill in changes proposed in this fix)

## How is this patch tested?

(Details)

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [x ] Projects
- [x ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [ ] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ x] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by AndreyBulezyuk** [comment]
It seems that i have not touched that part of mlflow but the pyspark tests failed.

**Comment by tomasatdatabricks** [comment]
There was a build issue that is fixed now. If you sync with master the tests should pass.

**Comment by AndreyBulezyuk** [comment]
Done. Please review @tomasatdatabricks

File: mlflow/projects/__init__.py [hunk 1]
```diff
@@ -812,7 +813,11 @@ def _get_local_artifact_cmd_and_envs(artifact_repo):
 
 def _get_s3_artifact_cmd_and_envs(artifact_repo):
     # pylint: disable=unused-argument
-    aws_path = posixpath.expanduser("~/.aws")
+    if platform.system() == "Windows":
+        win_user_dir = os.environ["userprofile"]
```
Comment (mparkhe) [inline mlflow/projects/__init__.py:817]: Use the [env variable in upper case](https://docs.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables): `os.environ["USERPROFILE"]`. Windows OS variables are case insensitive, but traditionally written as upper case.

**Review by mparkhe — COMMENTED** [review]
Thank you for the PR! Good change. One small recommended change

**Comment by mparkhe** [comment]
closes #2633

**Review by mparkhe — APPROVED** [review]

**Comment by mparkhe** [comment]
Awesome! Thank you @AndreyBulezyuk

**Pull request merged**